sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3217,30 +3217,16 @@ object GpuOverrides extends Logging {
             spark = TypeSig.STRING),
           ParamCheck(
             name = "from_base",
-            cudf = TypeSig.integral
-              .withAllLit()
-              .withInitialTypesPsNote("only values 10 and 16 are supported"),
+            cudf = TypeSig.integral,
```

Comment:
I know `integral` is already here, but I am wondering if it is correct, since Spark supports only Integer type.
See https://github.com/apache/spark/blob/v3.2.1/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala#L368
https://github.com/apache/spark/blob/v3.2.1/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala#L375

Reply:
Good catch!